wger/manager/views/workout.py

```diff
@@ -215,6 +223,77 @@ def add(request):
     return HttpResponseRedirect(workout.get_absolute_url())
 
 
+@login_required
+def export_workout(request):
+    '''
+    Export current workouts to a CSV file
+    '''
+    workouts = Workout.objects.filter(user=request.user)
+    response = HttpResponse(content_type='text/csv')
+    response['Content-Disposition'] = 'attachment; filename=mymodel.csv'
+    writer = csv.writer(response, csv.excel)
+    response.write(u'\ufeff'.encode('utf8'))
+    csv_header = ["Creation Date", "Comment", "Days", "Description", "Exercise"]
+    writer.writerow(csv_header)
+    for workout in workouts:
+        days = Day.objects.filter(training=workout.id)
+        if days:
+            for day in days:
+                workout_days = "-".join(
+                    [record.day_of_week for record in day.day.all()]
+                )
+                sets = Set.objects.filter(exerciseday=day.id)
+                if sets:
+                    for each_set in sets:
+                        exercises = "*".join(
+                            [exercise.name for exercise in each_set.exercises.all()]
+                        )
+                        writer.writerow([
+                            smart_str(workout.creation_date),
+                            smart_str(workout.comment),
+                            smart_str(workout_days),
+                            smart_str(day.description),
+                            smart_str(exercises)
+                        ])
+                else:
+                    writer.writerow([workout_days])
+        else:
+            writer.writerow([workout.creation_date])
+    return response
+
+
+@login_required
+def import_workout(request):
+    '''
+    Imports workouts from a CSV file into the database
+    '''
+    if request.POST and request.FILES:
+        csvfile = TextIOWrapper(
+            request.FILES['csv_file'].file,
+            encoding="utf-8"
+        )
+        reader = csv.DictReader(csvfile)
+        for row in reader:
+            workout = Workout(
+                creation_date=row["Creation Date"],
+                comment=row["Comment"],
+                user=request.user)
+            workout.save()
+            day = Day(training=workout, description=row["Description"])
+            day.save()
+            for day_name in row["Day"].split():
+                day.day.add(
+                    DaysOfWeek.objects.filter(day_of_week=day_name).first()
+                )
+            one_set = Set(exerciseday=day)
+            one_set.save()
+            for exercise in row["Exercise"].split("*"):
+                one_set.exercises.add(
+                    Exercise.objects.filter(name=exercise).first()
+                )
+    return HttpResponseRedirect(reverse('manager:workout:overview'))
```

Comment:
Wouldn't it make more sense to redirect to the newly imported workout? You can use the ``get_absolute_url`` method